demos/lexicon.txt

```diff
@@ -1,109 +1,271 @@
+accel
 ack
 acks
 addr
+aead
+aes
+aesni
+aka
+alloc
 alpn
+alt
 amazonrootca
 amazontrust
 amzn
 api
 apis
 appcallback
+arcfour
 argc
 args
 argv
+ascii
+asm
+asn
+asymetric
+async
+atttribute
 auth
+avoidng
 aws
+ba
 backoff
+backtrace
+badmac
 baltimore
 baltimorecybertrustroot
+bhargavan
+bignum
+bio
+bn
+boston
+bp
+bsd
 bufferlength
 buffersize
 bytesreceived
 bytessent
+bzero
 ca
 cacerts
+calloc
+cas
+cb
+cbc
+ccm
+ccs
 cdn
+cer
 cert
+certificatetemplate
 certs
+cfb
+chacha
+chachapoly
+chinese
```

Comment:
Woah is "chinese" not a dictionary word? lol

Reply:
Haha I did not notice that